src/features/auto-resize-grid/js/auto-resize.js

```diff
@@ -41,6 +41,7 @@
             if (newGridHeight !== prevGridHeight || newGridWidth !== prevGridWidth) {
               uiGridCtrl.grid.gridHeight = newGridHeight;
               uiGridCtrl.grid.gridWidth = newGridWidth;
+              uiGridCtrl.grid.api.core.raise.gridHeightChanged(prevGridHeight, newGridHeight);
```

Comment:
yeah, I would just update the event name.  

Are there other ways a gridDimension can get changed?   We need to make sure we raise the event everywhere it can occur.